src/main/java/frc/robot/commands/EntechCommandGroup.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import java.util.ArrayList;
+
+/**
+ *
+ * @author aryan
+ */
+public class EntechCommandGroup{
```

Comment:
this should be named entechcommandgroupbuilder, its confusing that its called a commandgroup when it is in fact never used as a command